client/components/payments-activity/payments-activity-data.tsx

```diff
@@ -57,41 +85,44 @@ const PaymentsActivityData: React.FC = () => {
 							content={ __( 'test charge content' ) }
 						/>
 					}
+					amount={ charges }
 					reportLink={ getAdminUrl( {
 						page: 'wc-admin',
 						path: '/payments/transactions',
 						filter: 'advanced',
 						type_is: 'charge',
 					} ) }
+					isLoading={ isLoading }
 				/>
 				<PaymentsDataTile
 					id="wcpay-payments-data-highlights__refunds"
 					label={ __( 'Refunds', 'woocommerce-payments' ) }
-					currencyCode="EUR"
-					amount={ 153200 }
+					currencyCode={ storeCurrency }
+					amount={ refunds }
 					reportLink={ getAdminUrl( {
```

Comment:
Date filter needs to be added to the link 

```suggestion
					reportLink={ getAdminUrl( {
						page: 'wc-admin',
						path: '/payments/transactions',
						filter: 'advanced',
						type_is: 'refund',
						'date_between[0]': moment(
							getDateRange().date_start
						).format( 'YYYY-MM-DD' ),
						'date_between[1]': moment(
							getDateRange().date_end
						).format( 'YYYY-MM-DD' ),
					} ) }
```